src/Appwrite/Platform/Workers/Migrations.php

```
@@ -363,57 +429,179 @@ function () use ($migration, $transfer, $projectDocument, $queueForRealtime) {
+                }
             }
         }
     }
+
+    protected function handleCSVExportComplete(Document $project, Document $migration, Mail $queueForMails): void
+    {
+        $options = $migration->getAttribute('options', []);
+        $bucketId = $options['bucketId'] ?? null;
+        $filename = $options['filename'] ?? 'export.csv';
+        $userInternalId = $options['userInternalId'] ?? '';
+        $resourceId = $migration->getAttribute('resourceId');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resourceId'. (undefined)

(UnusedLocalVariable)